buidl/hd.py

```diff
@@ -480,6 +481,22 @@ def from_mnemonic(
             seed, network=network, priv_version=priv_version, pub_version=pub_version
         ).traverse(path)
 
+    @classmethod
+    def from_dice(
+        cls,
+        rolls,
+        password=b"",
+        path="m",
+        network="mainnet",
+        priv_version=None,
+        pub_version=None,
+    ):
+        """Returns an HDPrivateKey object from a string of dice rolls (>=99 rolls recommended)"""
```

Comment:
We need some protection for non-cryptographically inclined developers. [From coldcard](https://coldcard.com/docs/verifying-dice-roll-math/"), "for 128-bit security, which we consider the absolute minimum, you need 50 rolls, and for 256-bits of security, 99 rolls"

I think the best move here is to add a keyword argument `allow_low_entropy=False` to this method signature. If `True`, then you can have entropy below the desired limit. If `False`, then you must have 50-99 rolls (depending on mnemonic length) or it will throw an error.